applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_windkessel_outlet_process.py

```diff
@@ -0,0 +1,138 @@
+import math
+import KratosMultiphysics
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsBiomedicalApplication as KratosBio
+
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return ApplyWindkesselOutletProcess(Model, settings["Parameters"])
+
+
+class ApplyWindkesselOutletProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings ):
+
+        KratosMultiphysics.Process.__init__(self)
+
+        default_settings = KratosMultiphysics.Parameters("""
+        {
+            "mesh_id"            : 0,
+            "model_part_name"    : "",
+            "variable_name"      : "PRESSURE",
+            "constrained"        : true,
+            "value"              : 0.0,
+            "interval"           : [0.0,"End"],
+            "resistance1"        : 0.0,
+            "resistance2"        : 0.0,
+            "compliance"         : 0.0,
+            "venous_pressure"    : 0.0,
+            "initial_pressure"   : 0.0,
+            "pressure_in_mmHg"   : true,
```

Comment:
Instead, I'd suggest a `pressure_unit` field that can take string values as `Pa` and `mmHg` (otherwise an error is thrown). By doing so you leave the room open to extend it to other units if required in the future.